config.ru

```diff
@@ -8,58 +8,25 @@ conn = PG::Connection.open(ENV["DATABASE_URL"])
 table = convert_keys(conn.exec('SELECT * from accounts'))
 
 app = -> (env) do
-  status = "<h1> Status account</h1>
+  filling = []
+  table.each do |line|
+    filling << "<tr>"
+   line.each do |key, val|
+      filling << "<td>#{line[key]}</td>"
+    end
+    filling << "</tr>"
+  end
```

Comment:
avoid each if you need a result:

```ruby
accounts = table.map do |line|
  tds = line.map { |key, val| "<td>#{val}</td>" }.join
  "<tr>#{tds}</tr>"
end
```
`filling` - is a bad name for variable as doesn't describe what it means.